agent/agent.go

```diff
@@ -414,8 +414,8 @@ func (a *Agent) runTaskTimeoutCommands(ctx context.Context, tc *taskContext) {
 	}
 	if taskGroup.Timeout != nil {
 		err := a.runCommands(ctx, tc, taskGroup.Timeout.List(), runCommandsOptions{})
-		tc.logger.Execution().ErrorWhenf(err != nil, "Error running timeout command: %v", err)
-		tc.logger.Task().InfoWhenf(err == nil, "Finished running timeout commands in %v.", time.Since(start).String())
+		tc.logger.Execution().ErrorWhen(err != nil, fmt.Sprintf("Error running timeout command: %v", err))
+		tc.logger.Task().InfoWhen(err == nil, fmt.Sprintf("Finished running timeout commands in %v.", time.Since(start).String()))
```

Comment:
these should really be message.WrapError(err, message.Fieldds{}) in most cases. 

There's also message types for Sprintf and Sptrintln, in general (which have lazy processing of the strings, rather than using fmt which gets interpolated a call time.